### ISSUE TITLE
[Request]

### ISSUE THREAD

**Issue by jtesse** [h5p/h5p-course-presentation#15]
It would be nice to be able to concatenate several project.

**Comment by falcon-git** [comment]
I'm not sure what you mean here?

**Comment by jtesse** [comment]
Sorry,
If I have a "course presentation" with 4 slides and another with 5 slides, I'd combine them to get a course presentation with 9 pages.

**Comment by falcon-git** [comment]
Oh :)

We will add copy paste to the next release and continue to improve on it. In the first release you'll be able to copy one and one item. It will in the future be possible to copy groups allowing you to copy all elements of a slide into another slide at once.

**Comment by jtesse** [comment]
now, I want to work with several students simultaneously, each on a presentation. At the end I will wish to achieve a single presentation.
Will it be possible to copy and paste between different presentations?

**Comment by icc** [comment]
Yes, you'll also be able to copy between Interactive Videos and Presentations.

**Comment by jtesse** [comment]
Thank you :-D

**Comment by otacke** [comment]
@jtesse Copy&Paste was added years ago, so I assume this issue can now be closed. Please feel holler if you think that closing it was unjustified.
